schema/resolvers.js

```diff
@@ -119,7 +119,42 @@ const Resolvers = {
     deleteOption: (_, args, ctx) =>
       ctx.repositories.Option.remove(args.input.id),
     undeleteOption: (_, args, ctx) =>
-      ctx.repositories.Option.undelete(args.input.id)
+      ctx.repositories.Option.undelete(args.input.id),
+    createOtherAnswer: async (root, args, ctx) => {
+      const parentAnswer = await ctx.repositories.Answer.get(
+        args.input.parentAnswerId
+      );
+
+      if (!isNil(parentAnswer.otherAnswerId)) {
+        return parentAnswer;
+      }
```

Comment:
Do you think maybe we should throw in this case?

Reply:
I guess there's no harm in throwing here. I was originally going to make it a noop, but I guess the UI should prevent you from trying to add a second otherAnswer and if you invoke it via the API, then it makes more sense to throw I think.